Update Teleport doc with latest instructions

DrW3RK opened the pull request:
Include detailed explanation of how teleports work and the fees involved

![KSM-teleport](https://user-images.githubusercontent.com/86818441/148551821-82d35b3b-449a-4164-953e-70be19056c8b.png)

DrW3RK committed:
Update Teleport doc with latest instructions

DrW3RK committed:
KSM to Statemine Teleport

gilescope commented on docs/learn/learn-teleport.md:
```diff
@@ -14,25 +14,34 @@ and [SPREE modules][], which together ensure that assets are not lost or duplica
 
 ## How to Teleport
 
+Here are a couple of things that you need to understand before submitting a teleport transaction
```
```suggestion
Here are a couple of things that you need to understand before submitting a teleport transaction:
```

gilescope commented on docs/learn/learn-teleport.md:
```diff
@@ -14,25 +14,34 @@ and [SPREE modules][], which together ensure that assets are not lost or duplica
 
+Here are a couple of things that you need to understand before submitting a teleport transaction
+
+- There will be a transaction fee imposed on the source chain. This fee is **not** automatically deducted from the teleport amount you plan to send. Your account balance should be greater than the teleport amount and the transaction fees. If you do not account for the sending fee, the teleport transaction will fail.
```
will fail (but you will not have been charged). <-- I presume?

profradha replied:
Yeah, that is correct! Will add that detail

DrW3RK committed:
Update the Title to be relevant

The tutorial is very specific and shows how to transfer native token across chains, but not the assets. A separate tutorial is needed.

gilescope commented on docs/learn/learn-teleport.md:
```diff
@@ -14,25 +14,34 @@ and [SPREE modules][], which together ensure that assets are not lost or duplica
+
+- There will be a transaction fee imposed on the source chain. This fee is **not** automatically deducted from the teleport amount you plan to send. Your account balance should be greater than the teleport amount and the transaction fees. If you do not account for the sending fee, the teleport transaction will fail.
+
+- There will be a transaction fee imposed at the destination chain. This fee is automatically deducted from the teleport amount you send. After this fee is deducted, the remaining account balance on the destination chain should be greater than its Existential Deposit. If not, the teleport transaction will fail.
```
will fail (but you will have been charged in both locations?)

profradha replied:
I could test again and confirm. As far as I can recollect, It did not charge me the receiving fee

gilescope commented on docs/learn/learn-teleport.md:
```diff
@@ -9,30 +9,51 @@ slug: ../learn-teleport
+- There will be a transaction fee imposed at the destination chain. This fee is automatically
+  deducted from the teleport amount you send. After this fee is deducted, the remaining account
+  balance on the destination chain should be greater than its Existential Deposit. If not, the
+  teleport transaction will fail.
```
can you say in both cases if the failed transaction will have tx fees charged. I am guessing there can be no charge in the first case but you will be charged both tx fees on the remote failure?

profradha replied:
Will test and get back to you. I will tag you once the PR is ready for review

gilescope reviewed (approved):
LGTM ( I have done said teleport )

DrW3RK commented:
Let me quickly verify the transaction fee situation and update the content to reflect it

DrW3RK committed:
More detail about failed transactions

DrW3RK commented:
@gilescope You are right! If the transaction fails at the receiving end, you would lose both the sending fee and the teleported  amount. The doc is now updated with those details. Merging this PR

gilescope commented:
Thanks for finding out - this rounds the docs off nicely.

On Fri, 7 Jan 2022 at 16:24, Radha ***@***.***> wrote:

> Merged #2942 <https://github.com/w3f/polkadot-wiki/pull/2942> into master.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/w3f/polkadot-wiki/pull/2942#event-5858755737>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGEJCHAS6E3KZCIMIVBYLTUU4HUNANCNFSM5LOZ2EMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
